### DISCUSSION TITLE
leafmap import error

### DISCUSSION THREAD

**Discussion by Farid-git7** [giswqs/geog-312#99] [Q&A] [answered]
![Screenshot 2024-10-07 121939](https://github.com/user-attachments/assets/779176d1-74d5-41a1-8956-2531b206e6a4)
I am having trouble importing leafmap through jupyter lab, the error comes as shown in the attached screenshot. Please help. Thanks!

**Comment by giswqs** [comment, marked as answer]
Looks like a pandas installation problem. Try uv: https://youtu.be/-YnfmHBHkWQ?si=bUJXNhFrMB3qcsMf
